### PR TITLE
GPII-2802 - Add permissions section to gcp/README

### PR DESCRIPTION
**Pull request by amatas**
This section describes how the permissions are set in GCP

### CODE REVIEW

**Comment by stepanstipl** [comment]
lgtm. We need to make sure this stays up to date re. the work @natarajaya's doing on IAM permissions.

File: gcp/README.md [hunk 1]
```diff
@@ -42,6 +42,16 @@ The DNS zones are:
 
 The environments that run in GCP need some initial resources that must be created by an administrator first. The [common part of this repository](../common) has the code and the instructions to do so.
 
+## Permissions
+
+The permissions in this project are set in three different levels: at organization level, at project level and at resource level.
+
+At the organization level we have the group _cloud-admin@raisingthefloor.org_ which contains the list of users that will manage the projects of the organization and has the high level permissions to do so. Also we have a SA dedicated for the project creation and the billing association: _projectowner@gpii-common-prd.iam.gserviceaccount.com_. This SA only has the enough permissions to create projects, assciate them to the billing account and create the IAMs needed in such project to allow the owner to create the resources inside it. The SA _projectowner@gpii2test-common-stg.iam.gserviceaccount.com_ must also be in the organization level, as the billing account is associated to this organization and it is used to attach it to the testing organization _test1.gpii.net_.
```
Comment (mrtyler) [inline gcp/README.md]: Please spell out the acronym SA the first time you use it.

Comment (mrtyler) [inline gcp/README.md]: Does this new arrangement require any changes to our team onboarding documentation?

File: gcp/README.md [hunk 2]
```diff
@@ -42,6 +42,16 @@ The DNS zones are:
...
+
+Each project has a SA which performs almost all the actions over the resources of such project. This SA only has the permissions needed to deploy GPII cloud. This SA doesn't have permissions to make changes outside the project which owns it.
+
+The permissions set at resource level are automaticallly set by Terraform in order to work properly among the rest of the components of the GPII cloud.
```
Comment (mrtyler) [inline gcp/README.md]: I don't understand this. Maybe an example would help?

File: gcp/README.md (hunk 1 above)
Comment (amatas) [reply gcp/README.md]: It's done in the onboarding and offboarding documentation.

**Comment by mrtyler** [comment]
LGTM

**Comment by natarajaya** [comment]
LGTM

**Pull request merged**